variables.tf

```diff
@@ -53,6 +65,24 @@ variable "name_prefix" {
   description = "prefix used in resource names"
 }
 
+variable "log_rotate_bytes" {
+  type        = string
+  default     = "250M"
+  description = "250 MB default for sized_log_rotation defaults."
+}
+
+variable "log_rotate_duration" {
+  type        = string
+  default     = "24H"
+  description = "24Hour default for timed_log_rotation."
+}
+
+variable "log_rotate_max_files" {
+  type        = string
+  default     = "100"
+  description = "100 Log files recommended for time based rotation. 32 recommended for date based rotation."
```

Comment:
nit: consider using `time-based` and `date-based`

Reply:
```suggestion
  description = "100 Log files recommended for time-based rotation. 32 recommended for date-based rotation."
```